Octokit.Tests.Integration/Clients/AssigneesClientTests.cs

```diff
@@ -4,16 +4,19 @@
 using Octokit.Tests.Integration;
 using Octokit.Tests.Integration.Helpers;
 using Xunit;
+using System.Collections.Generic;
 
 public class AssigneesClientTests
 {
     readonly IGitHubClient _github;
     readonly RepositoryContext _context;
+    private readonly IIssuesClient _issuesClient;
```

Comment:
:lipstick: drop the `private` modifier here (it's a pet peeve of mine, but the other fields also lack this)